include/aspect/material_model/drucker_prager.h

```diff
@@ -166,17 +140,23 @@ namespace aspect
          */
 
       private:
+        /**
+         * computes the volume fractions of the compositional fields, and returns it so that the sum
+         * of the volume fractions is one.
+         */
+        std::vector<double> compute_volume_fractions( const std::vector<double> &compositional_fields) const;
```

Comment:
you are right, this comes from code when there where different compositions present. I will remove this.